changelogs/fragments/3951-scaleway_compute_add_project_id.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - scaleway_compute - add posibility to use project identifier (new ``project`` option) instead of deprecated organization identifier (https://github.com/ansible-collections/community.general/pull/3951).
```

Sorry, in my last suggestion I overlooked that `s` needs to be there twice :)
```suggestion
  - scaleway_compute - add possibility to use project identifier (new ``project`` option) instead of deprecated organization identifier (https://github.com/ansible-collections/community.general/pull/3951).
```
Also, can you please rebase to get rid of the merge commit?